tests/core/test_farmer_harvester_rpc.py

```diff
@@ -414,6 +416,9 @@ async def test_farmer_get_harvester_plots_endpoints(
     reverse: bool,
     expected_plot_count: int,
 ) -> None:
+    if platform.system() == "Windows":
+        # Windows has an issue on Github CI with crashing workers
+        return
```

Comment:
Decorate with `@pytest.mark.skipif(sys.platform == "win32", reason="avoiding crashes on windows until we fix this").  Or whatever message.